### PR TITLE
Added metadata to pull requests

### LINKED ISSUE #271: GitHub comments must show that they are linked to a gruop
If we post a comment on a group there is no indication on GitHub that it actually was a group comment. We must add some indication that it was a group comment and filter these mentions out in the frontend.

### PR DESCRIPTION
**Pull request by TimvdLippe**
The metadata is added if the user wants to, as can be seen with this one on GitHub.

Fixes #271

---

This pull request was created on [preview-code.com](https://preview-code.com/preview-code/rite-evaluation/pulls/328/overview)

### CODE REVIEW

File: src/projects/create-pull-request/create-pull-request-description.html [hunk 1]
```diff
@@ -58,12 +60,17 @@
...
+        margin-top: 10px;
+      }
     </style>
 
     <paper-input class="title" label="Enter a pull request title" value="{{title}}"></paper-input>
     <markdown-input
       label="Enter a description for your pull request. Markdown is supported."
       markdown="{{description}}"></markdown-input>
+    <paper-checkbox id="metadata" checked="{{metadata}}">Add metadata to description</paper-checkbox>
+    <paper-tooltip for="metadata" offset="5">Metadata will include a direct link to the pull request on preview-code.com</paper-tooltip>
```
Comment (txsmith) [inline src/projects/create-pull-request/create-pull-request-description.html]: I think the checkbox and description can be a little bit clearer. For example: `Add direct link on GitHub` and as description: `Include a link back to the pull request on preview-code.com inside the pull description on GitHub`

Comment (TimvdLippe) [inline src/projects/create-pull-request/create-pull-request-description.html]: Fixed :+1:

**Comment by eanker** [comment]
LGTM

**Comment by eanker** [comment]
Accidentally closed this pr instead of merging.
All the checks had already passed.

**Pull request merged**